src/traces/cone/convert.js

```diff
@@ -107,6 +108,9 @@ function convert(scene, trace) {
     meshData.fresnel = trace.lighting.fresnel;
     meshData.opacity = trace.opacity;
 
+    // stash autorange pad value
+    trace._pad = anchor2coneSpan[trace.anchor] * meshData.vectorScale * trace._normMax * trace.sizeref;
```

Comment:
Awesome, I think we have a winner now! I don't see any cases (As I change the cone spacing, cone count in each direction, and vector length) where this fails to encompass all the cones, nor any where the padding looks unpleasantly large. 🏆